doc/api/net.md

```diff
@@ -515,56 +544,74 @@ changes:
     description: The `hints` option is supported now.
 -->
 
-Opens the connection for a given socket.
-
-For TCP sockets, `options` argument should be an object which specifies:
-
-  - `port`: Port the client should connect to (Required).
-
-  - `host`: Host the client should connect to. Defaults to `'localhost'`.
+* `options` {Object}
+* `connectListener` {Function} Common parameters of [`socket.connect()`][]
+  methods. Will be added as a listener for the [`'connect'`][] event once.
+* Returns: {net.Socket} The socket itself.
 
-  - `localAddress`: Local interface to bind to for network connections.
+Initiate the connection on a given socket.
 
-  - `localPort`: Local port to bind to for network connections.
+Note: this method is similar to a [connect(2)][] call, and before that,
```

Comment:
The original docs says

> Local interface to bind to for network connections.

(Minus the "interface" part because Node.js doesn't support binding to "eth0" or whatnot, it has to be a valid IP).
I think pointing out `bind(2)` might make it a bit more clearer that Node.js doesn't do `SO_BINDTODEVICE` here (or correct me if I've mistaken?). The `getaddrinfo(3)` part just tries to complete the whole options->syscall thing.